packages/lib/server/getRoutedUrl.ts

```diff
@@ -184,8 +186,8 @@ const _getRoutedUrl = async (context: Pick<GetServerSidePropsContext, "query" |
               stringify({ ...paramsToBeForwardedAsIs, "cal.action": "eventTypeRedirectUrl" })
             ),
             teamMembersMatchingAttributeLogic,
-            // formResponseId is guaranteed to be set because in catch block of trpc request we return from the function and otherwise it would have been set
-            formResponseId: formResponseId!,
+            formResponseId: formResponseId ?? null,
```

Comment:
Now either of `formResponseId` or `queuedFormResponseId` could be set.

@joeauyeung I created separate prop for queuedFormResponseId now because there types are different now. One is string and another is number